src/harness/testcases/WINNF_FT_S_EPR_testcase.py

```diff
@@ -0,0 +1,364 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import json
+import os
+import sas
+import sas_testcase
+from sas_test_harness import SasTestHarnessServer, generateCbsdRecords, \
+    generatePpaRecords
+from util import winnforum_testcase, configurable_testcase, writeConfig, \
+  loadConfig, makePpaAndPalRecordsConsistent
+
+
+class EscProtectionTestcase(sas_testcase.SasTestCase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  def generate_EPR_1_default_config(self, filename):
+    """ Generates the WinnForum configuration for EPR.1. """
+
+    # Load ESC record
+    esc_record_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'esc_sensor_record_0.json')))
+
+    # Load devices info
+    device_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    # Moving device_1(Cat A) to a location within 40 meters of ESC sensor
```

Comment:
When moving devices within/outside a certain radius from the ESC: slightly prefer doing it this way, so its clear that its with respect to the esc_record's lat / long  

device_1['installationParam']['latitude'] = esc_record_1['installationParam']['latitude'] + 0.55
device_1['installationParam']['longitude'] = esc_record_1['installationParam']['longitude'] + 0.05

(same for EPR.2)